js/ui/list/ui.list.edit.strategy.grouped.js

```diff
@@ -162,22 +162,24 @@ var GroupedEditStrategy = EditStrategy.inherit({
             var itemMeta = getItemMeta(this._collectionWidget.option("items")) ||
                            getItemMeta(this._collectionWidget.option("selectedItems"));
 
-            var groupKey = itemMeta.groupKey;
-            var item = itemMeta.item;
-
-            var selectedGroup;
-            $.each(result, function(_, item) {
-                if(item.key === groupKey) {
-                    selectedGroup = item;
-                    return false;
+            if(itemMeta) {
```

Comment:
if(!itemMeta) return

Reply:
And where is the test for this case?

Reply:
Ok, I will improve this. 
The test is crashed with itemMeta = undefined and so I added the "if" operator for the itemMeta variable.